app/controllers/drafts/ViewAllDraftMovementsController.scala

```diff
@@ -88,7 +87,7 @@ class ViewAllDraftMovementsController @Inject()(mcc: MessagesControllerComponent
         Redirect(routes.ViewAllDraftMovementsController.onPageLoad(ern, searchOptions.copy(index = 1)))
       } else {
         status(view(
-          form = form.fill(searchOptions),
+          form = form,
```

Comment:
The form is now cleared when successful. Copying from ViewAllMovementsController:

```scala
// stuff before
        val formToRender = if (form.hasErrors) form else form.fill(searchOptions)

        status(view(
          form = formToRender,
// stuff after
```